pennylane/beta/tapes/qnode.py

```diff
@@ -165,7 +165,7 @@ def get_tape(device, interface, diff_method="best"):
             return QNode._validate_device_method(device, interface)
 
         if diff_method == "parameter-shift":
-            return QNode._get_parameter_shift_method(device, interface)
+            return QNode._get_parameter_shift_tape(device), interface, "analytic"

```

Comment:
This ensures that if the user requests specifically the parameter-shift rule, the jacobian will be computed _only_ in analytic mode. Any parameter that does not support parameter-shift will raise an error.